docs/sources/enterprise/recorded-queries.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Recorded queries"
+description = "Grafana Enterprise recorded queries"
+keywords = ["grafana", "query", "queries", "recorded"]
+weight = 300
++++
+
+# Recorded queries
+
+Recorded queries allow you to see trends over time by taking a snapshot of a data point on a set interval. This can give you insight into historic trends.
+
+For our plugins that do not return time series, it might be useful to plot historical data. For example, you might want to query ServiceNow to see a history of request response times but it can only return current point-in-time metrics.
+
+## How recorded queries work
+
+### Note: An administrator must configure a Prometheus datasource and associate it with a [Remote write target](#remote-write-target) before recorded queries can be used.
+
+Recorded queries only work with backend data source plugins. You can recorded three types of queries:
+
+- single row and column - A query that returns a single row and column.
+- row count - A query that returns meaningful rows to be counted.
+- expression - Any expression. To learn more about creating and using expressions, see [Expressions](https://grafana.com/docs/grafana/latest/panels/expressions/).
+
+After a recorded query is created or enabled, it immediately creates a snapshot and continues to create snapshots at the set interval. The recorded query stops taking snapshots when it is disabled, deleted, or when Grafana is not running. Data points are gathered in the backend by running the recorded query and forwarding each result to a remote-write enabled Prometheus instance.
+
+## Using a recorded query
+
+To use a recorded query you’ll need to create one and add it to a dashboard. After that, it can be managed in **Preferences** from the **Recorded queries** tab.
```

Comment:
```suggestion
To use a recorded query, create one and add it to a dashboard. After that, it can be managed in **Preferences** from the **Recorded queries** tab.
```